### ISSUE TITLE
Wrong lib files in 3.4.2 Win Pack

### ISSUE THREAD

**Issue by flemingtech** [opencv/opencv#12209]
Extracted 3.4.2 via the WinPack link below, and tried to use it from VS 2017. However it wouldn't link, complaining that it couldn't open opencv_world**341**.lib.  I finally just had to rename the files in \build\x64\vc15\lib folder to *341.lib instead of *342.lib.

This was from a 100% fresh extraction, so somewhere the library name wasn't updated to 352 from 351.

https://sourceforge.net/projects/opencvlibrary/files/opencv-win/3.4.2/opencv-3.4.2-vc14_vc15.exe/download

**Comment by alalek** [comment]
> 3.4.2
> it couldn't open opencv_world341.lib

This is correct behavior. OpenCV 3.4.2 doesn't have "341" files.
You have stalled build configuration and must update it.

You should build from scratch (clean build folder).


Build / usage questions should go to Users OpenCV Q/A forum: http://answers.opencv.org

**Comment by flemingtech** [comment]
I must be confused, forgive me.

I've never had 3.4.1 in my life. I also have NOT tried to build opencv 3.4.2, so there was nothing to cause a 'stalled build configuration'.

1) I extracted the WinPack 3.4.2 to c:\opencv
2) In VS I added the following include paths to my project
C:\OpenCv\build\include;C:\OpenCv\build\include\opencv2;C:\OpenCv\build\include\opencv2\imgproc;
3) In VS I added the following lib path to MY linker library directory.
C:\opencv\build\x64\vc15\lib;
4) Compiled, and the result was an error from the linker that it was expecting 3.4.1.

I'm not sure how that is "correct behavior".
